internal/balancer/gracefulswitch/gracefulswitch.go

```diff
@@ -67,6 +67,8 @@ type Balancer struct {
 	// balancerCurrent before the UpdateSubConnState is called on the
 	// balancerCurrent.
 	currentMu sync.Mutex
+
+	pendingSwaps sync.WaitGroup
```

Comment:
Can we name this something else? This is actually waiting for the swap, but for the swapped-out balancer to be closed. I can't think of something nice. If you can't come up with something nicer either, then a comment would be useful. Thanks.